react/components/ProductPrice/index.js

```diff
@@ -155,21 +156,32 @@ const ProductPrice = (props, context) => {
               <IOMessage id={labelListPrice} />
             </div>
           )}
-          <Price
-            showPriceRange={showListPriceRange}
-            priceRange={listPriceRange}
-            price={listPrice}
-            rangeContainerClasses={classNames(
-              productPrice.listPriceValue,
-              listPriceRangeClass,
-              handles.price_listPriceRange,
-            )}
-            singleContainerClasses={classNames(
-              productPrice.listPriceValue,
-              listPriceClass,
-              handles.price_listPrice
-            )}
-          />
+          {!savingsAsPercentage &&
+            <Price
+              showPriceRange={showListPriceRange}
+              priceRange={listPriceRange}
+              price={listPrice}
+              rangeContainerClasses={classNames(
+                productPrice.listPriceValue,
+                listPriceRangeClass,
+                handles.price_listPriceRange,
+              )}
+              singleContainerClasses={classNames(
+                productPrice.listPriceValue,
+                listPriceClass,
+                handles.price_listPrice
+              )}
+            />}
+          {savingsAsPercentage && (listPrice - sellingPrice > 0) &&
+            <div
+              className={classNames(
+                productPrice.listPriceLabel,
+                handles.price_listPriceLabel,
+                'dib ph2 t-small-ns t-mini'
+              )}
+            >
+              {((listPrice - sellingPrice) / 1000) * 100}%
```

Comment:
Percentages varies on how it is rendered based on locale. You can read more about this here: https://phrase.com/blog/posts/number-localization/#toc_5

To make it global, use FormattedNumber from `react-intl`:
```suggestion
              <FormattedNumber value={((listPrice - sellingPrice) / 1000) * 100} style="percent" />
```